src/main/frontend/fs/sync.cljs

```diff
@@ -2745,27 +2745,22 @@
         (when-some [sm (sync-manager-singleton current-user-uuid graph-uuid
                                                (config/get-repo-dir repo) repo
                                                txid *sync-state)]
-          ;; 1. if remote graph has been deleted, clear graphs-txid.edn
-          ;; 2. if graphs-txid.edn's content isn't [user-uuid graph-uuid txid], clear it
-          (if (not (and user-uuid graph-uuid txid))
```

Comment:
This has been checked on https://github.com/logseq/logseq/blob/a8899e16c993614c0f36b9251b67be2bf926c44e/src/main/frontend/fs/sync.cljs#L2741.